### PR TITLE
added quantile estimator with parameters alpha and beta

### PR DESCRIPTION
**Pull request by lungben**
Currently, the quantile estimator definition 7 of
Hyndman, R.J and Fan, Y. (1996) "Sample Quantiles in Statistical Packages",
The American Statistician, Vol. 50, No. 4, pp. 361-365
is used for quantile calculation.

I suggest to add a generic quantile estimator method with alpha and beta as free parameters, so that the definitions 4-9 in the paper above can be used.

The following code gives the correct results (tested against SciPy), but has slightly worse performance than the current implementation.
However, thanks to dispatch, the existing implementation is used if alpha and beta are not given, therefore there should be no performance impact to existing code.

### CODE REVIEW

File: src/Statistics.jl [hunk 1]
```diff
@@ -968,6 +968,60 @@ quantile(itr, p; sorted::Bool=false) = quantile!(collect(itr), p, sorted=sorted)
...
+    require_one_based_indexing(v)
+
+    if !sorted
+        sort!(v)
+    end
+    n = length(v)
+    m = α + p * (one(α) - α - β)
+    aleph = (n * p + m)
+    k = min(max(Int(trunc(aleph)), 1), n - 1)
+    gamma = min(max(aleph - k, zero(aleph)), one(aleph))
```
Comment (nalimilan) [inline src/Statistics.jl]: `gamma = clamp(aleph - k, 0, 1)` should do the trick (arguments are promoted to a common type so there's no type instability). You could also use greek letters. Same for `k`.

File: test/runtests.jl [hunk 2]
```diff
@@ -534,6 +534,98 @@ end
     y = zeros(3)
     @test quantile!(y, x, [0.1, 0.5, 0.9]) === y
     @test y == [1.2, 2.0, 2.8]
+
+    #tests for quantile calculation with configurable α and β parameters
+    let v = [2, 3, 4, 6, 9, 2, 6, 2, 21, 17]
```
Comment (nalimilan) [inline test/runtests.jl]: No need for let (or use a new testset if you want to separate this block).

File: test/runtests.jl [hunk 3]
```diff
@@ -534,6 +534,98 @@ end
...
+        @test quantile(v, 0.8, 0.8, 1.0) ≈ 10.280000000000001
+        @test quantile(v, 0.8, 1.0, 0.0) ≈ 17.0
+        @test quantile(v, 0.8, 1.0, 0.2) ≈ 15.719999999999999
+        @test quantile(v, 0.8, 1.0, 0.4) ≈ 14.439999999999998
+        @test quantile(v, 0.8, 1.0, 0.6) ≈ 13.159999999999997
+        @test quantile(v, 0.8, 1.0, 0.8) ≈ 11.879999999999995
+        @test quantile(v, 0.8, 1.0, 1.0) ≈ 10.599999999999994
+        @test quantile(v, 1.0, 0.0, 0.0) ≈ 21.0
+        @test quantile(v, 1.0, 1.0, 1.0) ≈ 21.0
+
```
Comment (nalimilan) [inline test/runtests.jl]: Remove empty lines.

File: src/Statistics.jl [hunk 4]
```diff
@@ -968,6 +968,60 @@ quantile(itr, p; sorted::Bool=false) = quantile!(collect(itr), p, sorted=sorted)
...
+*The American Statistician*, Vol. 50, No. 4, pp. 361-365
+"""
+function quantile!(v:: AbstractVector, p:: Real, α:: Real, β:: Real; sorted:: Bool=false)
+    0 <= p <= 1 || throw(ArgumentError("input probability out of [0,1] range"))
+    0 <= α <= 1 || throw(ArgumentError("α parameter out of [0,1] range"))
+    0 <= β <= 1 || throw(ArgumentError("β parameter out of [0,1] range"))
+    require_one_based_indexing(v)
+
+    if !sorted
+        sort!(v)
```
[the remark this answers is not in the document]
Comment (nalimilan) [reply src/Statistics.jl]: Actually, maybe just use `_quantilesort!`. See how the existing `quantile!` does: with keyword arguments, you won't have to modify the logic.

**Comment by lungben** [comment]
Thanks for your suggestions. I am quite new at Julia and there is still a lot for me to learn.
I'll try to address your points in the next days.

File: src/Statistics.jl (hunk 1 above)
Comment (lungben) [reply src/Statistics.jl]: Good to know, thanks!

File: src/Statistics.jl (hunk 4 above)
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: I can use _quantilesort!, but I have to make the sort range slightly larger (max. 1 element at top and bottom) so that all elements relevant for all valid alpha and beta get sorted.

File: src/Statistics.jl [hunk 5]
```diff
@@ -968,6 +968,60 @@ quantile(itr, p; sorted::Bool=false) = quantile!(collect(itr), p, sorted=sorted)
...
+    quantile!(v:: AbstractVector, p:: Real, α:: Real, β:: Real; sorted:: Bool=false)
+
+Compute the quantile(s) of a vector `v` at a specified probability `p` on the interval [0,1].
+The keyword argument `sorted` indicates whether `v` can be assumed to be sorted; if
+`false` (the default), then the elements of `v` will be partially sorted in-place.
+The parameters `α` and `β` correspond to the parameters in Hyndman and Fan (1996)
+and allow calculation of quantiles for all estimators given in Table 3 of this paper
+(definitions 4-9).
+
+This method corresponds to SciPy's scipy.stats.mstats.mquantiles method (but being a bit less generic).
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: Updated the doc using the Wiki link and included an overview of the typical parameter choices.

The new version will be integrated into the current Julia implementation, therefore it will support e.g. arrays of p-parameters, etc.

**Comment by nalimilan** [comment]
I don't see the changes. Are you sure you pushed them?

**Comment by lungben** [comment]
I just pushed them. Sorry, still got some issues in tests which I had to fix.
Could you please take a look if it is fine now?

File: test/runtests.jl (hunk 2 above)
Comment (lungben) [reply test/runtests.jl]: done

File: test/runtests.jl (hunk 3 above)
Comment (lungben) [reply test/runtests.jl]: done

File: src/Statistics.jl [hunk 6]
```diff
@@ -818,24 +818,43 @@ _median(v::AbstractArray{T}, ::Colon) where {T} = median!(copyto!(Array{T,1}(und
...
 
+The keyword parameters α and β correspond to the same parameters in Hyndman and Fan,
+setting them to different values allows to calculate quantiles with any of the methods 4-9
+defined in this paper:
+
+Def. 4: α=0, β=1
+Def. 5: α=0.5, β=0.5
+Def. 6: α=0, β=0 (Excel PERCENTILE.EXC, Python exclusive-method)
+Def. 7: α=1, β=1 (Julia default, Excel PERCENTILE.INC, Python inclusive-method)
+Def. 8: α=1//3, β=1//3
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: I see, updated

File: src/Statistics.jl [hunk 7]
```diff
@@ -909,64 +930,90 @@ function _quantilesort!(v::AbstractArray, sorted::Bool, minp::Real, maxp::Real)
...
+* Hyndman, R.J and Fan, Y. (1996) "Sample Quantiles in Statistical Packages",
   *The American Statistician*, Vol. 50, No. 4, pp. 361-365
 
+* https://en.m.wikipedia.org/wiki/Quantile
+
 # Examples
 ```jldoctest
 julia> quantile(0:20, 0.5)
 10.0
-
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: done

File: src/Statistics.jl [hunk 8]
```diff
@@ -909,64 +930,90 @@ function _quantilesort!(v::AbstractArray, sorted::Bool, minp::Real, maxp::Real)
...
-    lv = length(v)
-    f0 = (lv - 1)*p # 0-based interpolated index
-    t0 = trunc(f0)
-    h  = f0 - t0
-    i  = trunc(Int,t0) + 1
+    n = length(v)
 
-    a = v[i]
-    b = v[i + (h > 0)]
+    if α==1 && β==1 # kept the specific implementation to avoid changing results on the level of machine precision
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: removed specific implementation for alpha=beta=1.
Results slightly changed - tests updated to use \approx instead of 
equality to allow differences on machine precision level.

**Comment by lungben** [comment]
@nalimilan Thanks for your feedback and suggestions!
Is there anything more required from my side so that it can be merged?

File: test/runtests.jl [hunk 9]
```diff
@@ -533,7 +535,90 @@ end
...
-    @test y == [1.2, 2.0, 2.8]
+    @test y ≈ [1.2, 2.0, 2.8]
+
+    #tests for quantile calculation with configurable α and β parameters
+    v = [2, 3, 4, 6, 9, 2, 6, 2, 21, 17]
+
+    # tests against SciPy quantile method
+    @test quantile(v, 0.0, α=0.0, β=0.0) ≈ 2.0
+    @test quantile(v, 0.2, α=1.0, β=1.0) ≈ 2.0
+    @test quantile(v, 0.4, α=0.0, β=0.0) ≈ 3.4000000000000004
```
Comment (mschauer) [inline test/runtests.jl]:
```suggestion
    @test quantile(v, 0.4, α=0.0, β=0.0) ≈ 3.40
```
(etc., if the that precision is not needed)

Comment (lungben) [reply test/runtests.jl]: This is a strange effect resulting from automatic generation of test cases. I rounded them to 2 digits, the test cases are still successful, but look nicer. Thanks for the feedback!

File: src/Statistics.jl [hunk 10]
```diff
@@ -818,24 +818,42 @@ _median(v::AbstractArray{T}, ::Colon) where {T} = median!(copyto!(Array{T,1}(und
 # for now, use the R/S definition of quantile; may want variants later
 # see ?quantile in R -- this is type 7
 """
-    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false)
+    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false, α::Real=1, β::Real=α)
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: I changed the alpha and beta names to ASCII.
Regarding `alpha=1.0`, this breaks the test cases
```
    @test quantile(Any[1, 2, 3], Float16(0.5)) isa Float16
    @test quantile(Any[1, Float16(2), 3], Float16(0.5)) isa Float16
```
because the results are promoted to Float64.
To keep the result being a Float16, one would explicity have to set `alpha=Float16(1)` or `alpha=1`.
Is this an acceptable tradeoff? If yes I can change the corresponding test.

File: test/runtests.jl [hunk 11]
```diff
@@ -480,29 +481,30 @@ end
...
+    @test quantile([1,2,3,4],0.5) ≈ 2.5
+    @test quantile([1,2,3,4],[0.5]) ≈ [2.5]
+    @test quantile([1., 3],[.25,.5,.75])[2] ≈ median([1., 3])
+    @test quantile(100.0:-1.0:0.0, 0.0:0.1:1.0) ≈ 0.0:10.0:100.0
+    @test quantile(0.0:100.0, 0.0:0.1:1.0, sorted=true) ≈ 0.0:10.0:100.0
+    @test quantile(100f0:-1f0:0.0, 0.0:0.1:1.0) ≈ 0f0:10f0:100f0
     @test quantile([Inf,Inf],0.5) == Inf
     @test quantile([-Inf,1],0.5) == -Inf
-    @test quantile([0,1],1e-18) == 1e-18
+    # here it is required to introduce an absolute tolerance because the calculated value is 0 in the method with parameters α and β
```
[the remark this answers is not in the document]
Comment (lungben) [reply test/runtests.jl]: done

File: src/Statistics.jl [hunk 12]
```diff
@@ -963,10 +1001,10 @@ julia> quantile(skipmissing([1, 10, missing]), 0.5)
 5.5
 ```
 """
-quantile(itr, p; sorted::Bool=false) = quantile!(collect(itr), p, sorted=sorted)
+quantile(itr, p; sorted::Bool=false, α::Real=1, β::Real=α) = quantile!(collect(itr), p, sorted=sorted, α=α, β=β)
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: done

File: src/Statistics.jl [hunk 13]
```diff
@@ -909,45 +929,63 @@ function _quantilesort!(v::AbstractArray, sorted::Bool, minp::Real, maxp::Real)
...
 
-Quantiles are computed via linear interpolation between the points `((k-1)/(n-1), v[k])`,
-for `k = 1:n` where `n = length(itr)`. This corresponds to Definition 7 of Hyndman and Fan
-(1996), and is the same as the R default.
+Samples quantile are defined by `Q(p) = (1-γ)*x[j] + γ*x[j+1]`,
+where ``x[j]`` is the j-th order statistic, and `γ` is a function of
+`j = floor(n*p + m)`, `m = α + p*(1 - α - β)` and
+`g = n*p + m - j`.
+
+As default (α=1, β=α), quantiles are computed via linear interpolation between the points
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: done

File: src/Statistics.jl [hunk 14]
```diff
@@ -897,8 +915,10 @@ function _quantilesort!(v::AbstractArray, sorted::Bool, minp::Real, maxp::Real)
 
     if !sorted
         lv = length(v)
-        lo = floor(Int,1+minp*(lv-1))
-        hi = ceil(Int,1+maxp*(lv-1))
+        # if not α==1 && β==1, we need to increase the sort range a bit, otherwise we may get wrong results
```
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: yes, this was still refering to the old implementation where alpha==beta==0 was treated differently. Comment deleted.

File: src/Statistics.jl (hunk 10 above)
[the remark this answers is not in the document]
Comment (lungben) [reply src/Statistics.jl]: Change incorporated and the keyword parameter `alpha=1` added to test cases mentioned above.

File: src/Statistics.jl [hunk 15]
```diff
@@ -833,35 +833,31 @@ _median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
 
 _median(v::AbstractArray{T}, ::Colon) where {T} = median!(copyto!(Array{T,1}(undef, length(v)), v))
 
-# for now, use the R/S definition of quantile; may want variants later
-# see ?quantile in R -- this is type 7
 """
-    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false, α::Real=1, β::Real=α)
+    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false, alpha::Real=1., beta::Real=alpha)
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false, alpha::Real=1.0, beta::Real=alpha)
```

File: src/Statistics.jl [hunk 16]
```diff
@@ -833,35 +833,31 @@ _median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
...
 
-By default (`α = β = 1`), quantiles are computed via linear interpolation between the points
-`((k-1)/(n-1), v[k])`, for `k = 1:n` where `n = length(v)`. This corresponds to Definition 7
-of Hyndman and Fan (1996), and is the same as the R and NumPy default.
+As default (alpha=1, beta=alpha), quantiles are computed via linear interpolation between the points
+`((k-1)/(n-1), v[k])`, for `k = 1:n` where `n = length(itr)`. This corresponds to Definition 7
+ of Hyndman and Fan (1996), and is the same as the R and NumPy default.
 
-The keyword arguments `α` and `β` correspond to the same parameters as in Hyndman and Fan,
+The keyword parameters alpha and beta correspond to the same parameters in Hyndman and Fan,
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
The keyword arguments `alpha` and `beta` correspond to the same parameters in Hyndman and Fan,
```

File: src/Statistics.jl [hunk 17]
```diff
@@ -833,35 +833,31 @@ _median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
...
-where `x[j]` is the j-th order statistic, and `γ` is a function of
-`j = floor(n*p + m)`, `m = α + p*(1 - α - β)` and
+where ``x[j]`` is the j-th order statistic, and `γ` is a function of
+`j = floor(n*p + m)`, `m = alpha + p*(1 - alpha - beta)` and
 `g = n*p + m - j`.
 
-By default (`α = β = 1`), quantiles are computed via linear interpolation between the points
-`((k-1)/(n-1), v[k])`, for `k = 1:n` where `n = length(v)`. This corresponds to Definition 7
-of Hyndman and Fan (1996), and is the same as the R and NumPy default.
+As default (alpha=1, beta=alpha), quantiles are computed via linear interpolation between the points
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
By default (`alpha = beta = 1`), quantiles are computed via linear interpolation between the points
```

File: src/Statistics.jl [hunk 18]
```diff
@@ -833,35 +833,31 @@ _median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
...
+`((k-1)/(n-1), v[k])`, for `k = 1:n` where `n = length(itr)`. This corresponds to Definition 7
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
`((k-1)/(n-1), v[k])`, for `k = 1:n` where `n = length(v)`. This corresponds to Definition 7
```

File: src/Statistics.jl [hunk 19]
```diff
@@ -833,35 +833,31 @@ _median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
...
+ of Hyndman and Fan (1996), and is the same as the R and NumPy default.
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
of Hyndman and Fan (1996), and is the same as the R and NumPy default.
```

File: src/Statistics.jl [hunk 20]
```diff
@@ -972,30 +966,30 @@ end
 end
 
 """
-    quantile(itr, p; sorted=false, α::Real=1, β::Real=α)
+    quantile(itr, p; sorted=false, alpha::Real=1., beta::Real=alpha)
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
    quantile(itr, p; sorted=false, alpha::Real=1.0, beta::Real=alpha)
```

File: src/Statistics.jl [hunk 21]
```diff
@@ -972,30 +966,30 @@ end
...
 `itr` can be assumed to be sorted.
 
 Samples quantile are defined by `Q(p) = (1-γ)*x[j] + γ*x[j+1]`,
 where ``x[j]`` is the j-th order statistic, and `γ` is a function of
-`j = floor(n*p + m)`, `m = α + p*(1 - α - β)` and
+`j = floor(n*p + m)`, `m = alpha + p*(1 - alpha - beta)` and
 `g = n*p + m - j`.
 
-As default (α=1, β=α), quantiles are computed via linear interpolation between the points
+As default (alpha=1, beta=alpha), quantiles are computed via linear interpolation between the points
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
By default (`alpha = beta = 1`), quantiles are computed via linear interpolation between the points
```

File: src/Statistics.jl [hunk 22]
```diff
@@ -833,35 +833,31 @@ _median(v::AbstractArray, dims) = mapslices(median!, v, dims = dims)
...
 """
-    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false, α::Real=1, β::Real=α)
+    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false, alpha::Real=1., beta::Real=alpha)
 
-Compute the quantile(s) of a vector `v` at a specified probability or vector or tuple of
-probabilities `p` on the interval [0,1]. If `p` is a vector, an optional
-output array `q` may also be specified. (If not provided, a new output array is created.)
-The keyword argument `sorted` indicates whether `v` can be assumed to be sorted; if
-`false` (the default), then the elements of `v` will be partially sorted in-place.
+Compute the quantile(s) of a collection `itr` at a specified probability or vector or tuple of
```
Comment (nalimilan) [inline src/Statistics.jl]: This method only accepts `v::AbstractVector`, so keep the existing description.

File: src/Statistics.jl [hunk 23]
```diff
@@ -972,30 +966,30 @@ end
...
+`j = floor(n*p + m)`, `m = alpha + p*(1 - alpha - beta)` and
 `g = n*p + m - j`.
 
-As default (α=1, β=α), quantiles are computed via linear interpolation between the points
+As default (alpha=1, beta=alpha), quantiles are computed via linear interpolation between the points
 `((k-1)/(n-1), v[k])`, for `k = 1:n` where `n = length(itr)`. This corresponds to Definition 7
  of Hyndman and Fan (1996), and is the same as the R and NumPy default.
 
-The keyword parameters α and β correspond to the same parameters in Hyndman and Fan,
+The keyword parameters alpha and beta correspond to the same parameters in Hyndman and Fan,
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
The keyword arguments `alpha` and `beta` correspond to the same parameters in Hyndman and Fan,
```

File: test/runtests.jl [hunk 24]
```diff
@@ -747,8 +712,8 @@ end
     @test var(x) === 16.25
     @test var(y) === 65//4
     @test std(x) === sqrt(16.25)
-    @test quantile(x, 0.5)  === 3.0
-    @test quantile(x, 1//2) === 3//1
+    @test quantile(x, 0.5, alpha=1)  === 3.0
```
Comment (nalimilan) [inline test/runtests.jl]: Why is this needed?

File: src/Statistics.jl [hunk 25]
```diff
@@ -929,45 +943,63 @@ function _quantilesort!(v::AbstractArray, sorted::Bool, minp::Real, maxp::Real)
...
     require_one_based_indexing(v)
 
-    lv = length(v)
-    f0 = (lv - 1)*p # 0-based interpolated index
-    t0 = trunc(f0)
-    h  = f0 - t0
-    i  = trunc(Int,t0) + 1
+    n = length(v)
+    m = alpha + p * (one(alpha) - alpha - beta)
+    aleph = n*p + m
```
Comment (nalimilan) [inline src/Statistics.jl]:
```suggestion
    aleph = n*p + oftype(p, m)
```

File: test/runtests.jl [hunk 26]
```diff
@@ -511,12 +510,12 @@ end
...
     @test quantile([1, 2, 3, 4], Float64[]) isa Vector{Float64}
     @test quantile([1, 2, 3, 4], []) isa Vector{Any}
-    @test quantile([1, 2, 3, 4], [0, 1]) isa Vector{Int}
+    @test quantile([1, 2, 3, 4], [0, 1], alpha=1) isa Vector{Int}
 
     @test quantile(Any[1, 2, 3], 0.5) isa Float64
     @test quantile(Any[1, big(2), 3], 0.5) isa BigFloat
-    @test quantile(Any[1, 2, 3], Float16(0.5)) isa Float16
-    @test quantile(Any[1, Float16(2), 3], Float16(0.5)) isa Float16
+    @test quantile(Any[1, 2, 3], Float16(0.5), alpha=1) isa Float16 # default type of alpha is Float
```
Comment (nalimilan) [inline test/runtests.jl]: Hmm, I hadn't realized using `alpha=1.0` by default would change this. That's not OK as must be preserve compatibility. I've made a suggestion to ensure that the type of `alpha` and `beta` have no effect on the output type.

Comment (lungben) [reply test/runtests.jl]: Your solution using offtype works fine, I'll roll back the tests so that they do not need to include alpha=1 explicitly.

File: test/runtests.jl (hunk 24 above)
Comment (lungben) [reply test/runtests.jl]: Your solution using offtype works fine, I'll roll back the tests so that they do not need to include alpha=1 explicitly.

File: src/Statistics.jl (hunk 22 above)
Comment (lungben) [reply src/Statistics.jl]: Copy/paste error, sorry for that!
Is corrected now.

**Comment by lungben** [comment]
There is an error in doctests due to numerical precision:
`julia> y
│ 3-element Array{Float64,1}:
│  1.2
│  2.0
│  2.8
│ ```
│ 
│ Subexpression:
│ 
│ y
│ 
│ Evaluated output:
│ 
│ 3-element Array{Float64,1}:
│  1.2000000000000002
│  2.0
│  2.8000000000000003`

The \approx function cannot be used in doctests, right?
Alternatively, I could add something like `round(y,8)` in the doctests.

**Comment by nalimilan** [comment]
Can't you just update the text to reflect the exact value that is returned (even if it's not pretty)?

**Comment by lungben** [comment]
I updated the doctests.

**Comment by lungben** [comment]
Single Travis CI jobs keep failing due to connection timeouts.
Is there a way to restart just a single job? Or do I need to do a "dummy" commit to restart all jobs?

**Comment by lungben** [comment]
OK now it worked, Doctests are now also passing. Please let me know if you have any more points.

**Comment by nalimilan** [comment]
Thanks @lungben! If we don't tag a new release before Julia 1.5, feel free to remind me to so that the feature is available (ideally at some point it will be possible to use new versions of Statistics on older Julai versions just like a normal package, but for now it's not possible).

**Comment by lungben** [comment]
@nalimilan Hi, after Julia release 1.4, would it be the right time now to tag it for Julia Release 1.5?

**Comment by nalimilan** [comment]
The 1.5 feature freeze will normally happen on April 15th, so we still have a few more weeks, which could allow including more PRs in the new release. Ping me before that date if I forget.

**Comment by lungben** [comment]
Hi @nalimilan !
Sorry I forgot to ping you in time :-(
Is there still a chance to get it into Julia 1.5?

**Comment by nalimilan** [comment]
Don't worry, I've updated the module in Julia last Monday. :-)
